earth_enterprise/src/fusion/autoingest/plugins.src/Database.src

```diff
@@ -771,8 +771,12 @@ DatabaseAssetVersionImplD::CreateSubsampledTerrain(
     terrain_project->LoadInsetInfo(ClientTmeshTilespaceFlat,
                                   beginSSLevel, endSSLevel,
                                   terrainInsets);
-    const uint beginMinifyLevel  = StartTmeshLevel;
-    const uint endMinifyLevel    = terrainInsets[0].effectiveMaxLevel;
+    const uint beginMinifyLevel  = (MiscConfig::Instance().UseMinification
```

Comment:
Minor: this logic is repeated a number of times. It might be nice to put it in a function somehow. It's not that much logic, though, so maybe it's not useful.